src/main/webapp/resources/js/pages/sequence-files/components/OverRepresentedSequences.jsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file renders the OverRepresentedSequences component.
+ */
+
+import React, { useEffect, useState } from "react";
+import { Layout, Table, Typography } from "antd";
+import { SPACE_MD } from "../../../styles/spacing";
+import { grey1 } from "../../../styles/colors";
+import { TabPaneContent } from "../../../components/tabs/TabPaneContent";
+import { getOverRepresentedSequences } from "../../../apis/files/sequence-files";
+import {
+  seqObjId,
+  seqFileId
+} from "../fastqc-constants";
+import { Monospace } from "../../../components/typography";
+
+export default function OverRepresentedSequences() {
```

Comment:
Comments

Reply:
Added in 139845ae0399cc73ea9aa42936a8341d24590480